nixos/modules/services/web-servers/telnet.nix

```diff
@@ -0,0 +1,57 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.telnet;
+
+in
+{
+
+  options.services.telnet = {
+
+    enable = mkOption {
+      default = false;
+      type = types.bool;
+      description = ''
+        Enable telnetd.
+      '';
+    };
+
+    port = mkOption {
+      default = 12345;
+      type = types.int;
+      description = ''
+        Port on which to listen.
+      '';
+    };
+
+  };
+
+  config = mkIf config.services.telnet.enable {
+
+    networking.firewall.allowedUDPPorts =  [ cfg.port ];
+
+    users.extraUsers.telnetd.uid = config.ids.uids.telnetd;
+    users.extraGroups.telnetd.gid = config.ids.gids.telnetd;
```

Comment:
Why does telnet need static id?

Reply:
It doesn't I guess. I copied other modules but why do other services need a static id either ?

Reply:
They need them if they have a large amount of files that needs to be adjusted to to the right owner in case the static uid changes.

Reply:
Yes, but does telnet create any files of its own?

Reply:
not that I know